lib/tss/response_handlers.rb

```diff
@@ -16,4 +16,12 @@ def handle_get_response(response, path, root_name)
     else fail "TSS: 'GET #{path}' failed with status = #{response.code}."
     end
   end
+
+  def handle_put_response(response, path)
+    case response.code
+    when 200 then JSON.parse(response.body)
+    when 404 then nil
+    else fail "TSS: 'PUT #{path}' failed with status = #{response.code}."
+    end
+  end
```

Comment:
That makes sense, but we should use `201` if something successfully created only, so `200` for the `PATCH` requests.

(I can change the handle_post_response and the tests if you think its OK)